docs/stake/overview.md

```diff
@@ -0,0 +1,161 @@
+
+The reward for a Compute Unit in an epoch fluctuates around the target value. It's determined by the proportion of correct proofs submitted by the CU relative to the total number of correct proofs submitted in that epoch. This algorithm ensures that the rewards pool stays close to the target value in FLT, making smooth adjustments when necessary.
+
+_The rewards earned by a CU for submitting proofs are shared between the compute provider and the staker. This division is determined by the staking rate outlined in the Capacity Commitment terms set by the compute Provider created it._
+
+#### Rewards for Deals
+
+An active Capacity Commitment, in which Compute Units submit proofs that they're ready to serve users' workloads, can be rented and participate in **Deals** created by users. A Deal published by a developer specifies the required number of CUs to participate in the Deal.
+
+The compute marketplace smart contract automatically matches the **Deal** with providers whose capacity meets the specified conditions. CUs then transition from the **Capacity Commitment Proofs submission** mode to the **Deal** service mode.
+
+For the time spent computing resources in a Deal, CC's Staker **continues to earn rewards in FLT** - the target revenue of CUs in Capacity Commitment proofs submission mode in FLT equivalent, multiplied by the staking rate.
+
+This arrangement ensures that _stakers' rewards generate stable and predictable revenue_, making staking as risk-free as possible.
+
+#### Vesting
+
+:::info[**Important numbers**]
+
+**Duration of one vesting period:** 1 epoch (24 hours)
+
+**Total number of vesting periods:** 182 epochs (6 months)
+
+:::
+
+**FLT** rewards earned each epoch are unlocked (vested) over time to encourage long-term commitment to the Network and consistent value contribution.
+
+Rewards unlocked proportionally every epoch (24 hours). Each unlock releases a fraction of the rewards earned in that period. This process continues for six months with a daily unlock of 1/182 total rewards.
+
+Let's break this down with a simple example:
+
+1. In the 5th epoch, you earn 182 FLT.
+2. Starting from this moment, for the next 182 days (about 6 months):
+
+   - Every day (which is one epoch), 1 FLT from this reward becomes available for you to use.
+   - This continues until all 182 FLT are unlocked.
+
+3. Now, let's say in the 6th epoch, you earn another 182 FLT.
+4. The same process starts again for this new reward:
+   - Every day for the next 182 days, another 1 FLT from this second reward becomes available.
+
+This pattern continues for each epoch where you earn rewards. It's like having multiple timers running at once, each slowly releasing your earned FLT over time.
+
+#### Slashing
+
+:::info[**Important numbers**]
+
+Currently, the **slashing rate** for one failed epoch per compute Unit is set to **0 %** for the **first month** of the Mainnet. It will be increased in the future.
```

Comment:
probably should add the actual starting date

Reply:
it's still unknown